src/Orleans.TestingHost/TestClusterOptions.cs

```diff
@@ -97,6 +97,15 @@ public class TestClusterOptions
         /// </remarks>
         public bool UseInMemoryTransport { get; set; } = true;
 
+        /// <summary>
+        ///  Gets or sets a value indicating whether to use Unix socket as transport  for connecting silos and clients, instead of TCP.
+        ///  Ignored if <see cref="UseInMemoryTransport"/> is enabled.
+        /// </summary>
+        /// <remarks>
+        /// Defaults to <see langword="false"/>
+        /// </remarks>
+        public bool UseUnixSocketTransport { get; set; } = false;
```

Comment:
Perhaps this should be an enum instead, since this and UseInMemoryTransport are mutually exclusive.